spec/factories/user.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  factory :user do
+    name 'Test user'
+    role 'user'
+    email 'example@mail.com'
+    password 'passw_1234'
+
+    factory :user_admin do
```

Comment:
and please read about `sequence` rspec method that helps to generate random fields' values. when you need to create multiple objects of the same entity it is much more convenient than to come up with all those names by hand in your test cases